src/stockwatch/use_cases/importing.py

```diff
@@ -72,13 +72,29 @@ def _process_sell_transaction_row(
     isin: IsinStr,
     row: dict[str, str],
     exchanges: list[CurrencyExchange],
+    cash_settlements: list[CashSettlement],
 ) -> ShareTransaction:
     descr = row["Omschrijving"].split()
     key_index = descr.index("Verkoop")
     nr_stocks = float(descr[key_index + 1].replace(",", "."))
     curr = row["Mutatie"]
     price = Amount(float(descr[key_index + 3].replace(",", ".")), curr)
     amount = apply_exchange(transaction_datetime, nr_stocks * price, exchanges)
+    # process delistings:
+    # so far only a single example of delisting has been presented
+    # for this one, it holds that the amount related to the sell transaction representing
+    # the delisting is found in a cash_settlement (and that one was in USD)
+    if amount.value == 0.0 and "DELISTING:" in descr:
```

Comment:
Not sure if I really like this logic. So for determining a delisting we're now dependent on the amount not fitting in any of the `exchanges` (and thus the `apply_exchange` returning 0.0. This seems really fragile. (e.g. the delisting being in euro's, or the `exchanges` by accident fitting the delisting amount.

Reply:
To me it seems to make sense that if it is a delisting (`DELISTING:` in the description), then we have to find the amount in the cash_settlements, otherwise we can use the exchanges? so it would become:
```
if "DELISTING:" in descr:
    amount = the_settlement.amount
    # We could also do the `apply_exchange` here instead of on line 151.
else:
   amount = apply_exchange(transaction, nr_stocks ...)
```

Reply:
I have only one example. The two lines that matter are as follows:
15-03-2021,13:34,12-03-2021,VANGUARD GLBLMINVO,IE00BYYR0C64,Contante Verrekening Aandelen,,USD,"yyy",USD,"yyy",
15-03-2021,13:34,12-03-2021,VANGUARD GLBLMINVO,IE00BYYR0C64,DELISTING: Verkoop 300 @ 0 EUR,,EUR,"0,00",EUR,"xxxx",

So the line with delisting is 0 EUR for a start, not related to exchange. I included the test for 0 EUR because I do not know what it looks like if the stock would have been in EUR instead of USD.

I can remove the test for 0 eur from the condition, you think that is better?

Reply:
ok, let's just test for delisting

Reply:
And I modified to do only apply_exchanges for non-delisting rows